packages/lesswrong/components/posts/LWPostsItem.tsx

```diff
@@ -190,6 +190,7 @@ export const styles = defineStyles("LWPostsItem", (theme: ThemeType) => ({
     marginRight: theme.spacing.unit*1.5,
     zIndex: theme.zIndexes.postItemAuthor,
     flex: 1000,
+    minWidth: "fit-content",
```

Comment:
Is this safe outside of the feed?

Reply:
I introduced it because of a weird bug (usernames on post items getting shrunk to zero width) that arose somehow due to the refactor. Shoggoth's guess was a style injection order difference. It really didn't seem like I'd done anything to make this happen, but this fixed it.